cmake/BuildLuzer.cmake

```diff
@@ -0,0 +1,53 @@
+set(LUZER_DIR ${PROJECT_BINARY_DIR}/luzer)
+set(LUZER_BUILD_DIR ${LUZER_DIR}/build)
+set(LUZER_LIBRARY_PATH ${LUZER_BUILD_DIR}/luzer)
+list(APPEND LUZER_LIBRARIES
+    ${LUZER_LIBRARY_PATH}/luzer/luzer.so
+    ${LUZER_LIBRARY_PATH}/luzer/libcustom_mutator.so
+)
+
+include(ExternalProject)
+
+get_target_property(LUA_LIBRARIES_LOCATION ${LUA_LIBRARIES} LOCATION)
+
+ExternalProject_Add(bundled-luzer
+    GIT_REPOSITORY https://github.com/ligurio/luzer
+    GIT_TAG ac580ef807a8c64a07636db2f1c88a19c7411b4c
+    GIT_PROGRESS TRUE
+    GIT_SHALLOW FALSE
+    SOURCE_DIR ${LUZER_DIR}/source
+    BINARY_DIR ${LUZER_BUILD_DIR}
+    TMP_DIR ${LUZER_DIR}/tmp
+    STAMP_DIR ${LUZER_DIR}/stamp
+    CONFIGURE_COMMAND
+        ${CMAKE_COMMAND} -B <BINARY_DIR> -S <SOURCE_DIR>
+            -G ${CMAKE_GENERATOR} ${BENCHMARK_CMAKE_FLAGS}
+            -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+            -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+            -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+
+            -DLUA_INCLUDE_DIR=${LUA_INCLUDE_DIR}
+            -DLUA_LIBRARIES=${LUA_LIBRARIES_LOCATION}
```

Comment:
Should we also add `LUA_EXECUTABLE` here? It is used only for tests but just in case...

Reply:
Right, `LUA_EXECUTABLE` is needed for running luzer's tests, so `LUA_EXECUTABLE` is not needed for using as an external Lua library.